Content.Shared/Movement/Systems/SharedMobCollisionSystem.cs

```diff
@@ -334,6 +338,13 @@ protected bool HandleCollisions(Entity<MobCollisionComponent, PhysicsComponent>
             {
                 cancellableDirection += mobMovement;
             }
+
+            if (_xenosCancelMovement &&
+                userIsXeno &&
+                _marineQuery.HasComp(other))
```

Comment:
```suggestion
                _marineQuery.HasComp(other))
```
Might be better to hook this onto some other component instead of marines, incase you want to make some kind of thing (maybe synths) able to move xenos

Reply:
> Might be better to hook this onto some other component instead of marines, incase you want to make some kind of thing (maybe synths) able to move xenos

Synths of any kind cannot push Xenos, they can only grab and drag T1s and some T2s.
Being able to grab and drag them isn't affected by this PR as its handled by another component.

Reply:
> > Synths of any kind cannot push Xenos, they can only grab and drag T1s and some T2s. Being able to grab and drag them isn't affected by this PR as its handled by another component.
> 
> Do note that afaik xenos also cannot push synths if theyre standing still, which CM13 handles that with rules "Body blocking the target’s escape is also to be considered a violation of self-preservation programming."

am pathing through synths as a drone currently.

Reply:
I'm not talking about this from a balancing perspective, simply splitting it into components so it isn't hardcoded, this is to make it easier to make changes in the future.